packages/web-ui/src/ds/atoms/Modal/index.tsx

```diff
@@ -34,10 +36,16 @@ export function Modal({
   footer,
   title,
   description,
+  size = 'regular',
 }: ModalProps) {
   return (
     <Dialog open={open} defaultOpen={defaultOpen} onOpenChange={onOpenChange}>
-      <DialogContent className='sm:max-w-modal'>
```

Comment:
this class wasn't getting applied because DialogContent was doing nothing with the prop className